tests/installation/await_install.pm

```diff
@@ -137,14 +133,14 @@ sub run {
         $timeout -= 30;
         diag("left total await_install timeout: $timeout");
         if (!$ret) {
-            if (get_var('LIVECD') && $screenlock_previously_detected) {
```

Comment:
In case you think this flag is useless, let's remove it completely, as it was only usage of it.

Reply:
Removed.